test/views/add/PlatformOperatorAddedViewSpec.scala

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.add
+
+import base.ViewSpecBase
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import viewmodels.PlatformOperatorSummaryViewModel
+import views.html.add.PlatformOperatorAddedView
+
+class PlatformOperatorAddedViewSpec extends ViewSpecBase {
```

Comment:
In general we do not write tests for content. We thought that the benefits are not big enough considering the time pressure we had.
You could test the text in the controller tests and delete this one.
If we want to test content, it will be a huge work as we do not have such tests at the moment.
We can discuss this in the team and decide if we want to do that.

Reply:
content test added to controller test